app/views/rock_pebble_mailer/rock_accepted_message.txt.erb

```diff
@@ -0,0 +1,5 @@
+Paired
+<%= @message %>
+Sincerely,
+
+The Paired Team
```

Comment:
Because we re-use this in a couple places, can we make a partial to dry up the 3 txt.erb files?